content/en/docs/quick-start/enable-pluggable-components.md

```diff
@@ -59,17 +59,17 @@ If you adopt [All-in-one Installation](../../quick-start/all-in-one-on-linux/),
 
 ### Installing on Kubernetes
 
-When you install KubeSphere on Kubernetes, you need to download the file [cluster-configuration.yaml](https://github.com/kubesphere/ks-installer/releases/download/v3.0.0/cluster-configuration.yaml) for cluster setting. If you want to install pluggable components, do not use `kubectl apply -f` directly for this file.
+When you install KubeSphere on Kubernetes, you need to download the file [cluster-configuration.yaml](https://github.com/kubesphere/ks-installer/releases/download/v3.0.0/cluster-configuration.yaml) for cluster setting.
 
-1. In the tutorial of [Installing KubeSphere on Kubernetes](../../installing-on-kubernetes/introduction/overview/), you execute `kubectl apply -f` first for the file [kubesphere-installer.yaml](https://github.com/kubesphere/ks-installer/releases/download/v3.0.0/kubesphere-installer.yaml). After that, to enable pluggable components, create a local file cluster-configuration.yaml.
+1. To enable pluggable components you want to install, change `false` to `true` for `enabled` in the file cluster-configuration.yaml. Here is [an example file](https://github.com/kubesphere/ks-installer/blob/release-3.0/deploy/cluster-configuration.yaml) for your reference. Save the file after you finish..
 
 ```bash
 vi cluster-configuration.yaml
 ```
 
-2. Copy all the content in the file [cluster-configuration.yaml](https://github.com/kubesphere/ks-installer/releases/download/v3.0.0/cluster-configuration.yaml) and paste it to the local file just created.
-3. In this local cluster-configuration.yaml file, enable the pluggable components you want to install by changing `false` to `true` for `enabled`. Here is [an example file](https://github.com/kubesphere/ks-installer/blob/master/deploy/cluster-configuration.yaml) for your reference. Save the file after you finish.
-4. Execute the following command to start installation:
+2. In the tutorial of [Installing KubeSphere on Kubernetes](../../installing-on-kubernetes/introduction/overview/), you execute `kubectl apply -f` first for the installer file [kubesphere-installer.yaml](https://github.com/kubesphere/ks-installer/releases/download/v3.0.0/kubesphere-installer.yaml) which waits the file cluster-configuration.yaml to apply for continuing the installation.
```

Comment:
I am a little confused here. Users need to apply the file kubesphere-installer.yaml first. The explanation should go first (namely in the first step). Why did you put it in the second step? And the current order makes the command `vi cluster-configuration.yaml` redundant and strange here.

Reply:
This section explains how to deploy ks on an existing K8s cluster with optional components enabled. So prepare the configuration file should be the first step, then apply the two files almost at the same time. Otherwise, user will be very easy to make mistake by just applying the installer yaml file and waiting the installation process without applying the configuration file.

Reply:
It happens notable times from end users. They thought just applying the installer yaml file and after the process finished, then apply the configuration yaml file. But the first file installation will hang over there because of the configuration file missing.

Reply:
Is it OK? @Sherlock113